Gems/PhysX/Code/Editor/DebugDraw.cpp

```diff
@@ -694,7 +694,12 @@ namespace PhysX
             const float minYBounds = -(numRows * heightfieldShapeConfig.GetGridResolution().GetY()) / 2.0f;
 
             auto heights = heightfieldShapeConfig.GetSamples();
-            
+
+            if (heights.empty())
+            {
+                return;
+            }
```

Comment:
nit: This could be moved at the beginning of the function to avoid doing all the previous operations when there are no samples.